tests/fixtures/e2e_suite.go

```diff
@@ -185,25 +180,56 @@ func resetRepo() error {
 	for _, entry := range entries {
 		// clean out repo of all changes
 		if entry.IsDir() {
-			_, err = wt.Remove(entry.Name())
+			repoPath := filepath.Join(localPath, entry.Name())
+
+			// open local path to cloned git server at local/repo(num).git
+			repo, err := git.PlainOpen(repoPath)
 			if err != nil {
 				return err
 			}
-		}
-	}
 
-	_, err = wt.Commit("Cleaning out repo", &git.CommitOptions{})
-	if err != nil {
-		return err
-	}
+			// open worktree
+			wt, err := repo.Worktree()
+			if err != nil {
+				return err
+			}
 
-	// git push to remote
-	err = repo.Push(&git.PushOptions{
-		RemoteName: "origin",
-		Auth:       auth,
-	})
-	if err != nil {
-		return err
+			// find path to manifests in repo
+			pathes, err := os.ReadDir(repoPath)
+			if err != nil {
+				return err
+			}
+
+			for _, path := range pathes {
+				// clean out repo of all changes
+				if path.IsDir() {
+					_, err = wt.Remove(path.Name())
+					if err != nil {
+						return err
+					}
+				}
+			}
+
+			_, err = wt.Commit("Cleaning out repo", &git.CommitOptions{})
+			if err != nil {
+				return err
+			}
+
+			// git push to remote
+			err = repo.Push(&git.PushOptions{
+				RemoteName: "origin",
+				Auth:       auth,
+			})
+			if err != nil {
+				return err
+			}
+		}
 	}
+
 	return nil
 }
+
+// func (s *E2ESuite) resetNamespace() error {
```

Comment:
is this something to be removed or done later?

Reply:
Removed.